lisa/tools/nfs_client.py

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from lisa.executable import Tool
+from lisa.operating_system import SLES, Debian, Redhat
+from lisa.tools import Firewall, Mount
+from lisa.util import LisaException
+
+
+class NFSClient(Tool):
+    @property
+    def command(self) -> str:
+        return ""
+
+    def can_install(self) -> bool:
+        return True
+
+    def setup(
+        self,
+        server_ip: str,
+        server_shared_dir: str,
+        mount_dir: str,
+        protocol: str = "tcp",
+    ) -> None:
+        # run installation
+        self._install()
```

Comment:
The installed will be called automatically, it doesn't need to call it in other methods.

Reply:
The issue here is that there is no command for NFS so I have set it to empty string. This results in the call to install getting skipped. Please let me know if you have a better way to do so.

Reply:
It should be able to detect if nfs module is installed, like the `mount` or `lsmod`. The `_check_exists` can be overwritten to detect by any other command.